config/log4j.properties

```diff
@@ -83,7 +83,7 @@ log4j.additivity.kafka.controller=false
 log4j.logger.kafka.log.LogCleaner=INFO, cleanerAppender
 log4j.additivity.kafka.log.LogCleaner=false
 
-log4j.logger.state.change.logger=TRACE, stateChangeAppender
+log4j.logger.state.change.logger=INFO, stateChangeAppender
```

Comment:
We also have the following. Should we change all the logging in controller log to INFO too for consistency?

`log4j.logger.kafka.controller=TRACE, controllerAppender`

Reply:
Sounds good to me, will work on this

Reply:
Let's do that as a separate PR though.